components/service/glean/src/main/java/mozilla/components/service/glean/private/HistogramMetricBase.kt

```diff
@@ -8,7 +8,7 @@ package mozilla.components.service.glean.private
  * A common interface to be implemented by all the histogram-like metric types
  * supported by the Glean SDK.
  */
-interface HistogramBase {
+interface HistogramMetricBase {
```

Comment:
This would require a change in the Glean parsers, as this name is used in the generated GV adapter code. Can we leave this as it is or change the parsers accordingly?

Reply:
Good catch.  I'll update this over in https://github.com/mozilla/glean_parser/pull/85